tests/functional/settings_clients.js

```diff
@@ -49,38 +55,24 @@ define([
     },
 
     'device panel is not visible without query param': function () {
-      var self = this;
-      return FunctionalHelpers.openPage(this, SIGNIN_URL, '#fxa-signin-header')
-        .then(function () {
-          return FunctionalHelpers.fillOutSignIn(self, email, FIRST_PASSWORD);
-        })
-
-        .findByCssSelector('#fxa-settings-header')
-        .end()
+      return this.remote
+        .then(openPage(this, SIGNIN_URL, '#fxa-signin-header'))
+        .then(fillOutSignIn(this, email, FIRST_PASSWORD))
 
-        .then(FunctionalHelpers.noSuchElement(this, '#devices'));
+        .then(testElementExists('#fxa-settings-header'))
+        .then(noSuchElement(this, '#devices'));
     },
 
     'device and apps panel works with query param, same device': function () {
-      var self = this;
-      self.timeout = 90 * 1000;
+      this.timeout = 90 * 1000;
       var testDeviceId;
 
-      return FunctionalHelpers.openPage(this, SIGNIN_URL_DEVICE_LIST, '#fxa-signin-header')
-        .then(function () {
-          return FunctionalHelpers.fillOutSignIn(self, email, FIRST_PASSWORD);
-        })
-
-        .findByCssSelector('#fxa-settings-header')
-        .end()
+      return this.remote
+        .then(openPage(this, SIGNIN_URL_DEVICE_LIST, '#fxa-signin-header'))
+        .then(fillOutSignIn(this, email, FIRST_PASSWORD))
 
-        .findByCssSelector('#clients .settings-unit-stub button')
-          .click()
-        .end()
-
-        .findByCssSelector('.clients-refresh')
```

Comment:
This clients-refresh click didn't seem to have any tested side effect, so I removed it.